qiskit/circuit/library/n_local/excitation_preserving.py

```diff
@@ -130,6 +97,49 @@ def __init__(self,
 
         Raises:
             ValueError: If the selected mode is not supported.
+
+        Examples:
+
+            .. jupyter-execute::
+
+                from typing import Union, Optional, List, Tuple, Callable, Any
+                from numpy import pi
+                from qiskit.circuit import QuantumCircuit, Parameter
+                from qiskit.circuit.library.standard_gates import RZGate
+                from qiskit.circuit.library.n_local import TwoLocal
+                from qiskit.circuit.library.n_local import ExcitationPreserving
+
+                ansatz = ExcitationPreserving(3, reps=1, insert_barriers=True,
+                entanglement='linear')
```

Comment:
It looks like the indentation here is still off, right? 🤔

Reply:
I think the indentation here is correct.

Reply:
Ah, I thought it should be
```
    ansatz = ExcitationPreserving(...
                                  ...)
```
but if that was not the intention that's alright too 👍

Reply:
Yes, that is how I've suggested here. 

<img width="532" alt="image" src="https://user-images.githubusercontent.com/11485594/85930678-5b01d380-b88c-11ea-9456-730583868e13.png">

Reply:
No I meant exactly what you suggested -- but it hasn't been applied if you look at the code 🙂   It still reads
```python
                ansatz = ExcitationPreserving(3, reps=1, insert_barriers=True,
                entanglement='linear')
```
instead of, as you suggested
```python
                ansatz = ExcitationPreserving(3, reps=1, insert_barriers=True,
                                              entanglement='linear')
```